go/datas/database_test.go

```diff
@@ -98,7 +98,8 @@ func (suite *DatabaseSuite) TestReadWriteCachePersists() {
 	suite.NoError(err)
 }
 
-func (suite *DatabaseSuite) TestWriteRefToNonexistentValue() {
+// TODO: Re-enable as part of BUG 3180
+func (suite *DatabaseSuite) SkipTestWriteRefToNonexistentValue() {
```

Comment:
Actually, it needed to be tweaked a little too, as the failure now won't happen until a BatchStore Flush or a Database Commit()